src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/services/email/EmailSenderDisabledTest.java

```diff
@@ -3,22 +3,23 @@
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.context.ApplicationContext;
 import org.springframework.test.context.TestPropertySource;
+import uk.gov.hmcts.reform.blobrouter.reconciliation.service.ReconciliationMailService;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 @SpringBootTest
-@TestPropertySource(
-    properties = {
-        "spring.mail.host=false"
-    }
-)
+@TestPropertySource(properties = "spring.mail.host=false")
 public class EmailSenderDisabledTest {
 
     @Autowired
     private ApplicationContext context;
 
+    @MockBean
+    private ReconciliationMailService mailService;
```

Comment:
is this needed?

Reply:
yeah, `EmailSender` is a constructor argument in `ReconciliationMailService` and the mail service fails to load when the `spring.mail.host` config is disabled.

Reply:
This will be removed after merging #725 which creates bean for disabled email config